### PR TITLE
Add development container setup description to README

### PR DESCRIPTION
**Pull request by andreas-bauer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,18 @@ This package is built and tested using [Python 3.10.0](https://www.python.org/do
...
+
+You can develop inside a Docker container using a [pre-build image](https://hub.docker.com/r/andib/cira-dev) that contains all dependencies and the recommended classification and labeling models.
+
+For this setup, you need
+
+1. [Docker](https://www.docker.com)
+2. [Visual Studio Code](https://code.visualstudio.com)
+3. [Remote Development Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
+
+You can find detailed prerequisites for this development setup [here]( https://code.visualstudio.com/docs/remote/containers)
```
Comment (JulianFrattini) [inline README.md:35]: There seems to be an unnecessary whitespace before the link.

File: README.md [hunk 2]
```diff
@@ -22,6 +22,18 @@ This package is built and tested using [Python 3.10.0](https://www.python.org/do
...
+
 ## Tests
 
 Run all tests via`pytest`.
```
Comment (JulianFrattini) [inline README.md:39]: While we are at it: there seems to be a whitespace missing before `pytest`

**Review by JulianFrattini — CHANGES_REQUESTED** [review]
Overall very good, but for those unfamiliar with developing using docker containers, please add a sentence or two on the process after the setup (i.e., open in VS code, then "open in remote container" given that Docker is running, etc.).

File: README.md (hunk 2 above)
Comment (andreas-bauer) [reply README.md]: fixed

**Review by JulianFrattini — APPROVED** [review]
Looks great now!
